src/Integration/Helpers/SonarLintOutputLogger.cs

```diff
@@ -41,21 +41,32 @@ public class SonarLintOutputLogger : ILogger
 
         public void WriteLine(string message)
         {
-            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, message);
+            var prefixedMessage = AddPrefixIfVerboseLogging(message);
+            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, prefixedMessage);
         }
 
         public void WriteLine(string messageFormat, params object[] args)
         {
-            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, messageFormat, args);
+            var prefixedMessageFormat = AddPrefixIfVerboseLogging(messageFormat);
+            VsShellUtils.WriteToSonarLintOutputPane(this.serviceProvider, prefixedMessageFormat, args);
         }
 
         public void LogVerbose(string messageFormat, params object[] args)
         {
             if (sonarLintSettings.DaemonLogLevel == DaemonLogLevel.Verbose)
             {
                 var text = args.Length == 0 ? messageFormat : string.Format(messageFormat, args);
-                WriteLine("DEBUG: " + text);
+                WriteLine("[DEBUG] " + text);
```

Comment:
Miscellaneous: all of our other prefixes use square brackets, so I changed to use there here too.

Reply:
LogVerbose eventually calls WriteLine, which now calls AddPrefixIfVerboseLogging.

Took me some time to realize why we would still want [Debug]. I get it now. It's not immediately clear, but I don't have a better solution. 🤔

Reply:
Yep; the existing code is a bit messy and seems unnecessarily complicated (I'm not sure why we have two overloads for `WriteLine`), but I wanted to minimise the edits since this is a last-minute change.